package.json

```diff
@@ -32,7 +32,10 @@
     "test:backend:core": "cd backend/core && yarn test",
     "test:e2e:backend:core": "cd backend/core && yarn test:e2e",
     "test:apps": "concurrently \"yarn dev:backend\" \"yarn test:app:public\"",
-    "lint": "eslint '**/*.ts' '**/*.tsx' '**/*.js'"
+    "lint": "eslint '**/*.ts' '**/*.tsx' '**/*.js'",
+    "install:backend": "cd backend/core && yarn install && yarn db:reseed",
+    "install:all": "yarn install && yarn install:backend",
```

Comment:
I feel like `yarn install` has a specific meaning (pulling in a bunch of dependencies) and that might make "yarn install:all" somewhat confusing since it does much more. How would you feel about calling these e.g. "setup:backend" and "setup:everything"? There's probably a better name than setup...

Reply:
I thought about that, but it's basically just doing an install in 2 directories. The reseed might be unexpected, but I figured the colon differentiated it enough from the regular install without having to remember a separate, repo-specific command. What if I made install:all just run install in both directories and had a setup command that did install:all plus reseed?

Reply:
I changed the command so that `yarn install:all` just does `yarn install` in both directories. `yarn setup` will now do the `yarn install:all` plus reseed the database.